awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvList.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useCallback } from 'react';
+import { useLocation } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Card } from '@patternfly/react-core';
+
+import { OrganizationsAPI } from '../../../api';
+import { getQSConfig, parseQueryString } from '../../../util/qs';
+import useRequest from '../../../util/useRequest';
+import PaginatedDataList from '../../../components/PaginatedDataList';
+import DatalistToolbar from '../../../components/DataListToolbar';
+
+import OrganizationExecEnvListItem from './OrganizationExecEnvListItem';
+
+const QS_CONFIG = getQSConfig('organizations', {
+  page: 1,
+  page_size: 20,
+  order_by: 'image',
```

Comment:
```suggestion
  order_by: 'name',
```
Just to be consistent with the EE main list.